src/tool/subcommands/api_cmd.rs

```diff
@@ -28,6 +28,12 @@ pub enum ApiCommands {
         /// Snapshot input paths. Supports `.car`, `.car.zst`, and `.forest.car.zst`.
         #[arg()]
         snapshot_files: Vec<PathBuf>,
+        /// Test filter
```

Comment:
Could be more helpful - how do I, the user, use it?

Reply:
Described the flag in more detail.